src/pages/quickstart.tsx

```diff
@@ -159,6 +162,22 @@ export const QuickstartPage = () => {
                 </Button>
               </CardContent>
             </Card>
+            <Card className="hover:shadow-md transition-shadow rounded-md">
+              <CardHeader className="p-4 pb-0">
+                <CardTitle className="text-lg flex items-center justify-between">
+                  Circuit Json
+                  <TypeBadge type="json" className="ml-2" />
```

Comment:
the best badge is probably "module", although circuit json can contain a board, component OR just a footprint 🤷 

```suggestion
                  <TypeBadge type="module" className="ml-2" />
```